datanode/sqlstore/migrations/0025_market_data_add_funding_rate.sql

```diff
@@ -0,0 +1,83 @@
+-- +goose Up
+
+alter table market_data add column if not exists funding_rate numeric;
+alter table current_market_data add column if not exists funding_rate numeric;
```

Comment:
Can / should we make them both nullable, and default to NULL? would make it easily compatible for all existing future market.

Reply:
I mentioned this in my comment also.. I think returning null over the api would be better than an empty string which is what I would expect given the current schema change.

Reply:
The nullable and default null value here is already implicit in the declaration. The table is too big to set a not null default, the alter command would take forever to complete on archive nodes.